UnitsNet.Tests/QuantityTypeConverterTest.cs

```diff
@@ -0,0 +1,348 @@
+            ITypeDescriptorContext context = new TypeDescriptorContext("SomeMemberName", new Attribute[] { });
+            Length length = Length.FromMeters(1);
+
+            Assert.Equal("1 m", unitsNETTypeConverter.ConvertTo(length, value));
+        }
+
+        [Fact]
+        public void ConvertTo_GiveSomeValues_ConvertToString()
+        {
+            UnitsNetTypeConverter<Length> unitsNETTypeConverter = new UnitsNetTypeConverter<Length>();
+            ITypeDescriptorContext context = new TypeDescriptorContext("SomeMemberName", new Attribute[] { });
+            Length length = Length.FromMeters(1);
+
+            Assert.Equal("1 m", unitsNETTypeConverter.ConvertTo(length, typeof(string)));
+            Assert.Equal("1 m", unitsNETTypeConverter.ConvertTo(context, culture, length, typeof(string)));
+
+            context = new TypeDescriptorContext("SomeMemberName", new Attribute[]
+            {
+                new DefaultUnitAttribute(Units.LengthUnit.Centimeter),
+                new ConvertToUnitAttribute(Units.LengthUnit.Millimeter)
+            });
+
+            Assert.Equal("1 m", unitsNETTypeConverter.ConvertTo(length, typeof(string)));
+            Assert.Equal("1 m", unitsNETTypeConverter.ConvertTo(context, culture, length, typeof(string)));
+
+            context = new TypeDescriptorContext("SomeMemberName", new Attribute[]
+            {
+                new DefaultUnitAttribute(Units.LengthUnit.Centimeter),
+                new ConvertToUnitAttribute(Units.LengthUnit.Millimeter),
+                new DisplayAsUnitAttribute(Units.LengthUnit.Decimeter)
+            });
+
+            Assert.Equal("1 m", unitsNETTypeConverter.ConvertTo(length, typeof(string)));
+            Assert.Equal("10 dm", unitsNETTypeConverter.ConvertTo(context, culture, length, typeof(string)));
+        }
+
+        [Fact]
+        public void ConvertTo_TestDisplayAsFormating_ConvertToFormatedString()
+        {
+            UnitsNetTypeConverter<Length> unitsNETTypeConverter = new UnitsNetTypeConverter<Length>();
+            ITypeDescriptorContext context = new TypeDescriptorContext("SomeMemberName", new Attribute[] { });
+            Length length = Length.FromMeters(1);
+
+            context = new TypeDescriptorContext("SomeMemberName", new Attribute[]
+            {
+                new DisplayAsUnitAttribute(Units.LengthUnit.Decimeter)
+            });
+
+            Assert.Equal("10 dm", unitsNETTypeConverter.ConvertTo(context, culture, length, typeof(string)));
+
+            context = new TypeDescriptorContext("SomeMemberName", new Attribute[]
+            {
+                new DisplayAsUnitAttribute(Units.LengthUnit.Decimeter, "v")
+            });
+
+            Assert.Equal("10", unitsNETTypeConverter.ConvertTo(context, culture, length, typeof(string)));
+        }
+
+
+        [Fact]
+        public void WrongUnitTypeInAttribut_DefaultUnit()
```

Comment:
`ConvertFrom_GivenWrongDefaultUnit_ThrowsInvalidOperationException`.
Here too, I think we should deciced on what exception types to use for invalid inputs. I think ArgumentException is good here too, since the arguments have value combinations we don't like.